packages/theme-chalk/src/table.scss

```diff
@@ -176,6 +176,9 @@
       white-space: nowrap;
       min-width: 50px;
     }
+    .#{$namespace}-button:focus-visible {
+      outline-offset: -2px;
```

Comment:
This would make the button inside the table look different from the others.
![image](https://github.com/element-plus/element-plus/assets/10278227/dfd30927-b602-4f32-bb40-c9ec83464146)